examples/s/d/nn/xcase/realization-0/iter-0/rms/bin/export_faultroom_surfaces.py

```diff
@@ -0,0 +1,43 @@
+"""Export faultroom surfaces via dataio with metadata."""
+
+import logging
+from pathlib import Path
+
+import fmu.dataio as dataio
+from fmu.config import utilities as utils
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.WARNING)
+
+CFG = utils.yaml_load("../../fmuconfig/output/global_variables.yml")
+
+HORISONNAMES = CFG["rms"]["horizons"]["TOP_RES"]
+
+# if inside RMS
+RMS_POL_CATEGORY = "GL_faultlines_extract_postprocess"
```

Comment:
Perhaps confusing that this is pointing to `faultlines` which is not the FaultRoom-output. Suggest something like "MyFaultroomOutput" or similarly generic.

(What does "GL" mean?)

Reply:
Copy/paste of other example... will fix